components/org.wso2.carbon.config/src/test/java/org/wso2/carbon/config/utils/EnvironmentUtils.java

```diff
@@ -60,31 +60,52 @@ public static void setEnv(Map<String, String> newVariables) {
         Map<String, String> newenv = new HashMap<>();
```

Comment:
Lets rename this to setEnvironementVariables as this is a public method

Reply:
changed as requested